src/livekit/options.ts

```diff
@@ -17,8 +17,9 @@ const defaultLiveKitPublishOptions: TrackPublishDefaults = {
   videoSimulcastLayers: [VideoPresets.h180, VideoPresets.h216] as VideoPreset[],
   screenShareEncoding: ScreenSharePresets.h1080fps30.encoding,
   screenShareSimulcastLayers: [
-    new VideoPreset(1920, 1080, 1_500_000, 5, "medium"),
+    new VideoPreset(1920, 1080, 500_000, 5, "medium"),
```

Comment:
is the screensharing fixed if we remote the 5fps layer entirely?

Reply:
Sadly it only gets fixed if we remove all simulcast layers. Apparently, temporal layers don't work in Chrome or something like this.

But I think I get your point :) I.e. it's better to have screen sharing working in both browsers rather than having one browser with screen sharing support and another without.

I've removed them and only left the 30fps preset for the screen-sharing encoding. Now it should work in both browsers (but without a simulcast).

Reply:
where is the change with the removed screen sharing layers?

Reply:
I've forgotten to push, done :)